test_regress/t/t_hier_block_import.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2024 by Wilson Snyder. This program is free software; you can
```

Comment:
Can you please add a t_hier_block_import_cmake.py, referring to these same sources?